overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -50,13 +49,21 @@ type Monitor struct {
 	monitorStop   chan struct{}
 	netlinkErrors chan error
 	netlinkEvents chan netlink.UEvent
+
+	// devices lookup keeps track of all observed devices to know
+	// when to ignore a spurious event (in case it happens, e.g. when device gets reported by both
+	// the enumeration and monitor on startup).
+	// the lookup is based on device paths which are guaranteed to be unique and stable till device gets removed.
```

Comment:
Perhaps replace the word `lookup` with `key`?

Reply:
Done.